README.md

```diff
@@ -93,6 +74,41 @@ Each runs their respective process configured as either a `-controller` or an `-
   The on-host process responsible for publishing update metadata and executing
   update activities.
 
+## Getting Started
+
+### Label nodes
+
+To start Bottlerocket updater operator agent on nodes, you will need to add `bottlerocket.aws/updater-interface-version` label.
+We recommend using `update-interface-version` 2.0.0 for Bottlerocket OS version >=v0.4.1 which uses Bottlerocket's [update API](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/updater/README.md#update-api) to dispatch updates.
+
+With `kubectl` configured for the desired cluster, you can use the below command to get all nodes
```

Comment:
nit: Would it be worthwhile tossing a link in here to instructions for setting up kubectl for those who haven't done it yet?

Reply:
I can add a link to genereic `kubectl` overview page https://kubernetes.io/docs/reference/kubectl/overview/ . Does that sound good ?

Reply:
Yeah, I think that'd work just fine. Thanks!

Reply:
Nit: maybe a missing colon at the end before all the commands you suggested in this doc?

Reply:
Done.